crates/libm-cdylib/src/lib.rs

```diff
@@ -0,0 +1,156 @@
+#![allow(dead_code)]
+#![cfg_attr(not(test), feature(core_intrinsics, lang_items))]
+#![cfg_attr(not(test), no_std)]
+
+#[path = "../../../src/math/mod.rs"]
+mod libm;
```

Comment:
Could this do `extern crate libm` instead?

Reply:
That would pull in std, see below.